Voilà à peu près 10000 ans (C'est une approximation), une civilisation a laissé des connaissances qui m'interroge sur le sens de la lumière et de l'éther. Les scientifiques du 20 ième siècle ont abandonné l'éther car ils ne le comprenaient pas !

Voilà ma définition de l'éther : Impédance du vide =360 $\Omega$ et 1 $\Omega$= 1° degré 

In [6]:
show(LatexExpr(r"\pi \ naturel"))
show(LatexExpr(r"\pi^2 = \frac{8}{8.1}\times 10=3,14"))

\pi \ naturel

\pi^2 = \frac{8}{8.1}\times 10=3,14

In [14]:
%display latex
p=(8/8.1)*10;p

9.87654320987654

In [13]:
round(sqrt(p),2)

3.14

Un cercle de 360°=3.14, de diamètre 1

La notion de champ est une erreur fondamentale car elle escamote l'éther : Champ électrique E et magnétique M son orthogonale, parait il ? L'éther produit un angle de 45° qui transforme le E vertical en M horizontal, c'est un effet de miroir.

La dispersion rotatoire (Effet Cotton) est en fait la résonnance : Longueur d'onde $2\pi$, résonance $\pi^+ - \pi^-=0$ 